tox.ini

```diff
@@ -43,6 +43,8 @@ deps =
     cov: pytest-cov
     jwst: jwst[test] @ git+https://github.com/spacetelescope/jwst.git
     jwst: stpipe @ git+https://github.com/spacetelescope/stpipe.git
+    devdeps: astropy>=0.0.dev0
```

Comment:
This will just pull the very latest version of astropy from `PyPi` which I don't thing is the intention of this PR.

Astropy builds from source just fine in our environments so you should just point this at the astropy main branch